Mage.Sets/src/mage/cards/b/BreachTheMultiverse.java

```diff
@@ -0,0 +1,133 @@
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreatureOrPlaneswalkerCard;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.TargetCard;
+import mage.target.targetpointer.FixedTargets;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+public class BreachTheMultiverse extends CardImpl {
+    public BreachTheMultiverse(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{5}{B}{B}");
+
+        //Each player mills ten cards. For each player, choose a creature or planeswalker card in that player’s
+        //graveyard. Put those cards onto the battlefield under your control. Then each creature you control becomes a
+        //Phyrexian in addition to its other types.
+        this.getSpellAbility().addEffect(new MillCardsEachPlayerEffect(10, TargetController.EACH_PLAYER));
+        this.getSpellAbility().addEffect(new BreachTheMultiverseEffect());
+        this.getSpellAbility().addEffect(new BreachTheMultiverseSubtypeEffect());
+    }
+
+    private BreachTheMultiverse(final BreachTheMultiverse card) {
+        super(card);
+    }
+
+    @Override
+    public BreachTheMultiverse copy() {
+        return new BreachTheMultiverse(this);
+    }
+}
+
+class BreachTheMultiverseEffect extends OneShotEffect {
+
+    public BreachTheMultiverseEffect() {
+        super(Outcome.PutCreatureInPlay);
+        staticText = "for each player, choose a creature or planeswalker card in that player's graveyard. Put those " +
+                "cards onto the battlefield under your control.";
+    }
+
+    private BreachTheMultiverseEffect(final BreachTheMultiverseEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public BreachTheMultiverseEffect copy() {
+        return new BreachTheMultiverseEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        UUID controllerId = source.getControllerId();
+        Player controller = game.getPlayer(controllerId);
+        if (controller != null) {
+            Set<Card> cardsToBattlefield = new HashSet<>();
+            for (UUID playerId : game.getState().getPlayersInRange(controllerId, game)) {
+                Player player = game.getPlayer(playerId);
+                if (player != null) {
+                    boolean creatureOrPlaneswalkerInGraveyard = false;
+                    for (UUID cardId : player.getGraveyard()) {
+                        Card card = game.getCard(cardId);
+                        if (card != null && (card.isCreature(game) || card.isPlaneswalker(game))) {
+                            creatureOrPlaneswalkerInGraveyard = true;
+                            break;
+                        }
+                    }
```

Comment:
you can replace this whole loop with `player.getGraveyard().count` and a creature/walker filter